packages/web/src/common/store/account/sagas.js

```diff
@@ -186,6 +186,10 @@ export function* fetchAccountAsync({ isSignUp = false }) {
         reason: 'ACCOUNT_NOT_FOUND'
       })
     )
+    if (!isNativeMobile) {
```

Comment:
I know @rickyrombo was seeing some Sentry errors with localStorage recently on Safari mobile. Just wondering if it's safe to _always_ clear this regardless of whether we've actually set it and if there were any learnings from that issue we should apply here to fail safely if localStorage isn't accessible?

Reply:
Tested on ios safari. it seems ok. i thought about getting & then removing if it exists, but it's just strictly more work because the implementation of remove per spec is 

https://html.spec.whatwg.org/multipage/webstorage.html

---- 

The removeItem(key) method steps are:

If [this](https://webidl.spec.whatwg.org/#this)'s [map](https://html.spec.whatwg.org/multipage/webstorage.html#concept-storage-map)[key] does not [exist](https://infra.spec.whatwg.org/#map-exists), then return null.

Set oldValue to [this](https://webidl.spec.whatwg.org/#this)'s [map](https://html.spec.whatwg.org/multipage/webstorage.html#concept-storage-map)[key].

[Remove](https://infra.spec.whatwg.org/#map-remove) [this](https://webidl.spec.whatwg.org/#this)'s [map](https://html.spec.whatwg.org/multipage/webstorage.html#concept-storage-map)[key].

[Reorder](https://html.spec.whatwg.org/multipage/webstorage.html#concept-storage-reorder) [this](https://webidl.spec.whatwg.org/#this).

[Broadcast](https://html.spec.whatwg.org/multipage/webstorage.html#concept-storage-broadcast) [this](https://webidl.spec.whatwg.org/#this) with key, oldValue, and null.